src/main/java/net/fabricmc/fabric/mixin/events/tick/MixinPlayerEntity.java

```diff
@@ -0,0 +1,30 @@
+package net.fabricmc.fabric.mixin.events.tick;
+
+import net.fabricmc.fabric.api.event.player.EquipmentTickCallback;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.util.Identifier;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(PlayerEntity.class)
+public abstract class MixinPlayerEntity extends LivingEntity {
+	protected MixinPlayerEntity(EntityType<? extends LivingEntity> type, World world) {
+		super(type, world);
+	}
+
+	@Inject(method = "updateTurtleHelmet", at = @At("HEAD"))
```

Comment:
Any reason for injecting into `updateTurtleHelmet` wouldnt at the end of `update` be cleaner?

Reply:
Mainly for organization’s sake. It felt kinda wrong to me to not use the method that’s already there to tick equipment.